src/containers/Loaders/ArticleLoader.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import classes from '../../CSS/Loader.css';
+
+const ArticleLoader = () => (
+  <div className={classes.article_loader}>
+    <div />
+    <div />
+    <div />
+    <div />
```

Comment:
what are you doing here

Reply:
@geofrocker 
There I'm creating a custom animation that is shown until the request data has returned.
These divs are required by the articleLoader component to load this animation